components/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
+  <div class="ampstart-hero center">
+    <div class="ampstart-hero-img">
+      {{#images}}
+        {{#image}}
+          <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="responsive" media="{{media}}" alt="{{alt}}"></amp-img>
```

Comment:
Thanks for the tip, it's great that we can define multiple images for each device. Since wide and narrow have different dimensions, I was following the same approach as outlined in [fullpage-hero](https://github.com/ampproject/ampstart/blob/master/components/images/fullpage-hero.snip.html#L22).

Should we also look to update that example or do them together in a separate PR?